<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePersonWindow</class>
 <widget class="QDialog" name="ChangePersonWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  qlineargradient( spread:pad,  x1:0, y1:0, x2:1, y2:0,  stop:0 rgba(105, 57, 153, 100),  stop:1 rgba(165, 105, 189, 255));  border: 1px solid #5D3B9B;  border-radius: 5px;
font-family: Everlasting;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>311</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">backgroud-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;</string>
   </property>
   <widget class="QPushButton" name="Close_button_Clicked">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>630</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 12px;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Закрыть окно </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangePerson_button_Clicked">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>580</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 12px;
font-size: 10pt;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Изменить пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
border: none;
font-size: 12pt;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Изменение пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Adres">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="login">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Фамилия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Имя:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Адрес:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Логин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Пароль:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Пользователь успешно изменён!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeLName_button_Clicked">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
font-weight: bold;
border: none;
font-size: 10pt;
background-color: none;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Изменить фамилию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeFName_button_Clicked">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
font-weight: bold;
border: none;
font-size: 10pt;
background-color: none;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Изменить имя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeAdres_button_Clicked">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
font-weight: bold;
border: none;
font-size: 10pt;
background-color: none;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Изменить адрес</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeLogin_button_Clicked">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
font-weight: bold;
border: none;
font-size: 10pt;
background-color: none;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Изменить логин</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangePassword_button_Clicked">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
font-weight: bold;
border: none;
font-size: 10pt;
background-color: none;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Изменить пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="MName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeMName_button_Clicked">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
font-weight: bold;
border: none;
font-size: 10pt;
background-color: none;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Изменить отчество</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Отчество:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Phone">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangePhone_button_Clicked">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
font-weight: bold;
border: none;
font-size: 10pt;
background-color: none;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Изменить телефон</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Телефон:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Role">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
font-size: 10pt;
color: white;
}
QComboBox::Item {
background-color: rgba(255,255,255,40);
color:white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Покупатель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Менеджер</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Администратор</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Роль:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
